sargo/config.json

```diff
@@ -527,6 +527,79 @@
       "BoardConfigVendor": [],
       "device-vendor": [],
       "vendor-skip-files": []
+    },
+    "full": {
+      "overlays-dir": "",
+      "system-bytecode": [
+        "system/app/atfwd/atfwd.apk",
+        "system/app/ConferenceDialer/ConferenceDialer.apk",
+        "system/priv-app/GoogleDocumentsUIPrebuilt/GoogleDocumentsUIPrebuilt.apk",
+        "system/priv-app/GoogleExtServicesPrebuilt/GoogleExtServicesPrebuilt.apk",
+        "system/priv-app/GoogleNetworkPermissionConfig/GoogleNetworkPermissionConfig.apk",
+        "system/priv-app/GoogleNetworkStack/GoogleNetworkStack.apk",
+        "system/priv-app/GooglePackageInstaller/GooglePackageInstaller.apk",
+        "system/priv-app/GooglePermissionControllerPrebuilt/GooglePermissionControllerPrebuilt.apk",
+        "product/app/ModuleMetadataGooglePrebuilt/ModuleMetadataGooglePrebuilt.apk",
+        "product/app/DiagnosticsToolPrebuilt/DiagnosticsToolPrebuilt.apk::PRESIGNED",
+        "product/app/VZWAPNLib/VZWAPNLib.apk",
+        "product/priv-app/AmbientSensePrebuilt/AmbientSensePrebuilt.apk::PRESIGNED",
+        "product/priv-app/AppDirectedSMSService/AppDirectedSMSService.apk",
+        "product/priv-app/ConnMO/ConnMO.apk",
+        "product/priv-app/DCMO/DCMO.apk",
+        "product/priv-app/DiagMon/DiagMon.apk",
+        "product/priv-app/EuiccGoogle/EuiccGoogle.apk::PRESIGNED",
```

Comment:
OTOH, this is something that depends on a Google app - namely, GMS, which is what the intention of the full config was previously.

Reply:
Yes, I can only agree on that.